packages/@uppy/aws-s3/CHANGELOG.md

```diff
@@ -1,93 +1,157 @@
-# @uppy/aws-s3
+# @uppy/aws-s3-multipart
```

Comment:
To not confuse people, this should be `@uppy/aws-s3`.

All commits below will still reference `aws-s3-multipart` though. Alternatively a comment in the changelog explaining the naming change.